src/js/build.js

```diff
@@ -21,20 +21,7 @@ const header = `/*!
 
 const popperCDN = 'https://unpkg.com/popper.js/dist/umd/popper.js'
 const autoPrefixerConfig = {
-  cascade: false,
-  browsers: [
-    '>= 1%',
-    'last 1 major version',
-    'not dead',
-    'Chrome >= 45',
-    'Firefox >= 38',
-    'Edge >= 12',
-    'Explorer >= 10',
-    'iOS >= 9',
-    'Safari >= 9',
-    'Android >= 4.4',
-    'Opera >= 30'
-  ]
+  cascade: false
```

Comment:
Do not remove our browserlist it should follow Bootstrap browserlist

Reply:
Please check this: https://github.com/mobihack/bulma-customize/issues/6

Reply:
After updating all the dependencies and trying a build. That message was shown in the console log. I saw that the same list was in the file.

Reply:
you just have to rename `browsers` by `overrideBrowserslist` and it works fine 😉

Reply:
Yeah. It's just that we already have the file with the same content in the repo. Also the file can also be used by other tools when needed. Also https://github.com/browserslist/browserslist/issues/138#issuecomment-456064021 It will work with both ways. I will be happy with what you will find best :smiley:

Reply:
We cannot use files here, because autoprefixer is run in a browser

Reply:
Oh. I forgot about that.